### PR TITLE
Fix some bug on Windows

### PR DESCRIPTION
**Pull request by moeshin**

### CODE REVIEW

File: plugin/wakatime.vim [hunk 1]
```diff
@@ -38,10 +38,15 @@ let s:VERSION = '9.0.1'
...
     " Script Globals
-    let s:home = expand("$WAKATIME_HOME")
-    if s:home == '$WAKATIME_HOME'
+    let s:home = ''
+    if exists('$WAKATIME_HOME')
+        let s:home = expand('$WAKATIME_HOME')
+    else
         let s:home = expand("$HOME")
     endif
+    if has('win32')
```
Comment (alanhamlett) [inline plugin/wakatime.vim:47]: Would it hurt to always replace `\` with `/`, even when not on Windows?

File: plugin/wakatime.vim [hunk 2]
```diff
@@ -90,13 +95,16 @@ let s:VERSION = '9.0.1'
         " Detect os and architecture
         if s:IsWindows()
             let s:osname = "windows"
-            if has("win64")
+            if has("win64") || isdirectory(expand('$WINDIR') . '\SysWOW64')
```
Comment (alanhamlett) [inline plugin/wakatime.vim:98]: Let's be extra careful and do:

`if has("win64") || exists('$WINDIR') && isdirectory(expand('$WINDIR') . '\SysWOW64')`

**Review by alanhamlett — APPROVED** [review]
I'll make the changes from my comments after merging. Thanks!

File: plugin/wakatime.vim (hunk 1 above)
Comment (moeshin) [reply plugin/wakatime.vim:47]: Tested on Windows 11, Msys2 and WSL no problem.

File: plugin/wakatime.vim (hunk 2 above)
Comment (moeshin) [reply plugin/wakatime.vim:98]: Ok. I don't know if there is any other better way to detect 64-bit.

**Pull request merged**